config/environments/test.rb

```diff
@@ -58,4 +58,15 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  # set fake_data for pam, don't do real calls, just use fake data
+  if AppConfig.pam.enable?
+    Rpam2.fake_data =
+      {
+        usernames: Set["pam_user1", "pam_user2"],
+        servicenames: Set["pam_test", "pam_test_controlled"],
+        password: "123456",
+        env: { email: "me@example.com" }
```

Comment:
- Align the elements of a hash literal if they span more than one line.
- Space inside { detected.
- Space inside } detected.